bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/GiniBank.kt

```diff
@@ -106,7 +106,7 @@ object GiniBank {
     /**
      * Shows the return reasons dialog in the return assistant, if enabled.
      */
```

Comment:
[Request] Can you please add a sentence here that it's disabled by default? Thanks!